providers.tf

```diff
@@ -1,3 +1,12 @@
 provider "aws" {
-  region = "us-east-1"
-}
\ No newline at end of file
+  region  = "us-east-1"
```

Comment:
In order to store the terraform state in a provided S3 bucket, we must specifically configure the backend. I also am referencing a specific AWS profile name, so that when I run `terraform apply` locally, it will properly identify the credentials.